corporate_perspective/best_practices.md

```diff
@@ -16,11 +16,11 @@ title: BridgeAI MLOps Knowledge Hub
 
 Within best practices for MLOps, there are a number of other areas where principles must be considered in order to implement an optimal AI/ML offering. The following considerations can be expanded on by clicking on them:
 
-* [Objective and Metric Best Practices](#objective-and-metric-best-practices)
-* [Infrastructure Best Practices](#infrastructure-best-practices)
-* [Data Best Practices](#data-best-practices)
-* [Model Best Practices](#model-best-practices)
-* [Code Best Practices](#code-best-practices)
+* [Objective and Metric Best Practices](./corporate_perspective/best_practices.html#objective-and-metric-best-practices)
+* [Infrastructure Best Practices]./corporate_perspective/best_practices.html(#infrastructure-best-practices)
+* [Data Best Practices](./corporate_perspective/best_practices.html#data-best-practices)
+* [Model Best Practices](./corporate_perspective/best_practices.html#model-best-practices)
+* [Code Best Practices](./corporate_perspective/best_practices.html#code-best-practices)
```

Comment:
This looks wrong.  You are trying to refer to an anchor within the same document, so it should just be #blah rather than a relative path to another document and an anchor?

Reply:
You aren't wrong, though I've had to do this because I recently tested the table of contents for some of the pages on the live page and found they just redirect to the landing page rather than to the headings in the page. I think I need to approach the relative links setup differently

Reply:
How is the ToC being built?

Reply:
The way you mentioned before, i.e. #blah to make reference to headings in a given page. Though I've looked through the code for the default layouts and think that removing the base href line will resolve this. Trying now and reverting the ToC to the original method of reference.

Reply:
Pushed this now